package-lock.json

```diff
@@ -12056,8 +12056,8 @@
 			}
 		},
 		"del": {
-			"version": "git+https://github.com/sindresorhus/del.git#51662acf8c0b0632ed577fc2bb64a65a677ead9b",
-			"from": "git+https://github.com/sindresorhus/del.git#51662acf8c0b0632ed577fc2bb64a65a677ead9b",
+			"version": "git+https://github.com/sindresorhus/del.git#ca05c65ca582b47a20e1b64539dd5ef45a0e8419",
+			"from": "git+https://github.com/sindresorhus/del.git",
```

Comment:
I expected this to happen for the package-lock in this case because of all the dependency updates merged into master without any rebuilds (dependabot updates)